cmd/internal/cli/inspect_linux.go

```diff
@@ -65,44 +81,70 @@ func init() {
 	SingularityCmd.AddCommand(InspectCmd)
 }
 
-func getLabelsFile(appName string) string {
+func getPathPrefix(appName string) string {
 	if appName == "" {
-		return " cat /.singularity.d/labels.json;"
+		return "/.singularity.d"
 	}
+	return fmt.Sprintf("/scif/apps/%s/scif", appName)
+}
 
-	return fmt.Sprintf(" cat /scif/apps/%s/scif/labels.json;", appName)
+func getSingleFileCommand(file string, label string, appName string) string {
+	str := fmt.Sprintf(" if [ -f %s/%s ]; then", getPathPrefix(appName), file)
```

Comment:
you could use strings.Builder here instead.

Reply:
Thanks! Fixed.